Add CLI ocis decomposedfs check-treesize (#556)
owncloud/docs-ocis: modules/ROOT/pages/maintenance/commands/commands.adoc

In rare circumstances, it can be necessary to initiate indexing manually for a given space or user. Though the search service handles exception cases automatically, it can happen that the search service was not able to complete indexing due to a dirty shut-down of the service or because of a bug. Re-indexing should *only* be run on explicit request and supervision by ownCloud support. See the xref:{s-path}/search.adoc#manually-trigger-re-indexing-a-space[Manually Trigger Re-Indexing a Space] section at the _Search_ service for details.

== Resume Post-Processing

> If post-processing fails in one step due to an unforseen error, current uploads will not be retried automatically. Starting with Infinite Scale release 3.1, a system administrator can instead run a xref:{s-path}/postprocessing.adoc#resume-post-processing[CLI command] to retry the failed upload.

Reviewer: ```suggestion
If post-processing fails in one step due to an unforeseen error, current uploads will not be retried automatically. Starting with Infinite Scale release 3.1, a system administrator can instead run a xref:{s-path}/postprocessing.adoc#resume-post-processing[CLI command] to retry the failed upload.
```

== Inspect and Manipulate Node Metadata

> Infinite Scale provides a CLI command where you can inspect and manipulate node metadata.

Author: ```suggestion
Infinite Scale provides a CLI command with which you can inspect and manipulate node metadata.
```

   --help, -h              show help
----

== Inspect and Repair Node Tree Sizes

> Infinite Scale provides a CLI command where you can inspect and repair node tree sizes. This command can be necessary for the very rare case if there are Spaces or files shown in the WebUI where the size does not match reality. Note for the repair option that Infinite Scale must be shut down.

Reviewer: ```suggestion
Infinite Scale provides a CLI command with which you can inspect and repair node tree sizes. This command can be necessary in very rare cases where spaces or files are shown in the Web UI with a size not matching reality. For the repair option Infinite Scale must be shut down.
```
